test/e2e/specs/liquidation.spec.js

```diff
@@ -84,4 +84,51 @@ describe('Wallet App Test Cases', () => {
       cy.contains('105.52 IST').should('exist');
     });
   });
+
+  context('Place bids and make all vaults enter liquidation', () => {
+    it('should create a vault minting 400 ISTs and giving 80 ATOMs as collateral', () => {
+      cy.createVault({ wantMinted: 400, giveCollateral: 80, userType: 'gov1' });
+    });
+    it('should place bids from the CLI successfully', () => {
+      cy.switchWallet('gov1');
+      cy.addNewTokensFound();
+      cy.getTokenAmount('IST').then(initialTokenValue => {
+        cy.placeBidByPrice({
+          fromAddress: accountAddresses.gov1,
+          giveAmount: '90IST',
+          price: 9,
+        });
+
+        cy.placeBidByDiscount({
+          fromAddress: accountAddresses.gov1,
+          giveAmount: '80IST',
+          discount: 10,
+        });
+
+        cy.placeBidByDiscount({
+          fromAddress: accountAddresses.gov1,
+          giveAmount: '150IST',
+          discount: 15,
+        });
+
+        cy.getTokenAmount('IST').then(tokenValue => {
+          expect(tokenValue).to.lessThan(initialTokenValue);
+        });
+      });
+    });
+
+    it('should change ATOM price to 9.99', () => {
+      cy.setOraclePrice(9.99);
+    });
+
+    it('switch to user1 wallet', () => {
+      cy.switchWallet('user1');
+    });
+
+    it('should verify vaults that are at a risk of being liquidated', () => {
```

Comment:
changing the wallet and the price aren't being tested, right?
```suggestion
    it('should verify vaults that are at a risk of being liquidated', () => {
      cy.switchWallet('user1');
      cy.setOraclePrice(9.99);
```

Reply:
Makes sense. Changing it after testing it locally.

Reply:
Updated